app/screens/home/channel_list/servers/servers_list/index.tsx

```diff
@@ -41,9 +41,10 @@ const ServerList = ({servers}: Props) => {
         addNewServer(theme);
     }, [servers]);
 
-    const renderServer = useCallback(({item: t}: ListRenderItemInfo<ServersModel>) => {
+    const renderServer = useCallback(({item: t, index}: ListRenderItemInfo<ServersModel>) => {
         return (
             <ServerItem
+                highlight={index === 0}
```

Comment:
Just to make sure, why are we highlighting the server with index 0? Shouldn't we highlight the current one, or something like that?

Reply:
Yes, I thought about it but is essentially the same.. the tutorial highlight will only appear the first time you open the server switch modal and you'll only have one server in that moment.

Now let's consider the case when you have idk, let's say 5 servers and we just introduced this, say your active server is down in the list and maybe not even visible at as you'll need to scroll down, then this would be a disaster hehehe